components/sandbox/toolchain-host-operator/production/stone-prd-host1/miner-detector-cronjob.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: batch/v1
+kind: CronJob
```

Comment:
Kube linter check is failing, there are 7 issues in this PR.

```
kustomize build components/sandbox/toolchain-host-operator/production/stone-prd-host1/ | kube-linter lint -
KubeLinter v0.6.4-0-g4f0b1e01fd

<standard input>: (object: toolchain-host-operator/miner-detector batch/v1, Kind=CronJob) The container "find-miners" is using an invalid container image, "quay.io/codeready-toolchain/miner-detector:latest". Please use images that are not blocked by the `BlockList` criteria : [".*:(latest)$" "^[^:]*$" "(.*/[^:]+)$"] (check: latest-tag, remediation: Use a container image with a specific tag other than latest.)

<standard input>: (object: toolchain-host-operator/miner-detector batch/v1, Kind=CronJob) container "find-miners" does not have a read-only root file system (check: no-read-only-root-fs, remediation: Set readOnlyRootFilesystem to true in the container securityContext.)

<standard input>: (object: toolchain-host-operator/miner-detector batch/v1, Kind=CronJob) container "find-miners" is not set to runAsNonRoot (check: run-as-non-root, remediation: Set runAsUser to a non-zero number and runAsNonRoot to true in your pod or container securityContext. Refer to https://kubernetes.io/docs/tasks/configure-pod-container/security-context/ for details.)

<standard input>: (object: toolchain-host-operator/miner-detector batch/v1, Kind=CronJob) container "find-miners" has cpu request 0 (check: unset-cpu-requirements, remediation: Set CPU requests and limits for your container based on its requirements. Refer to https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits for details.)

<standard input>: (object: toolchain-host-operator/miner-detector batch/v1, Kind=CronJob) container "find-miners" has cpu limit 0 (check: unset-cpu-requirements, remediation: Set CPU requests and limits for your container based on its requirements. Refer to https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits for details.)

<standard input>: (object: toolchain-host-operator/miner-detector batch/v1, Kind=CronJob) container "find-miners" has memory request 0 (check: unset-memory-requirements, remediation: Set memory requests and limits for your container based on its requirements. Refer to https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits for details.)

<standard input>: (object: toolchain-host-operator/miner-detector batch/v1, Kind=CronJob) container "find-miners" has memory limit 0 (check: unset-memory-requirements, remediation: Set memory requests and limits for your container based on its requirements. Refer to https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits for details.)
```

Reply:
@hugares thanks for pointing that out. Are those checks mandatory ? 
I'm asking because I'm not 100% sure we can address those ( or at least not all of them immediately ) 

We are tagging only with `latest` , so this would required a change in our ci/cd pipeline --> 
<standard input>: (object: toolchain-host-operator/miner-detector batch/v1, Kind=CronJob) The container "find-miners" is using an invalid container image, "quay.io/codeready-toolchain/miner-detector:latest". Please use images that are not blocked by the `BlockList` criteria : [".*:(latest)$" "^[^:]*$" "(.*/[^:]+)$"] (check: latest-tag, remediation: Use a container image with a specific tag other than latest.)

Also this one I think it's needed since we write some files inside the container --> 
<standard input>: (object: toolchain-host-operator/miner-detector batch/v1, Kind=CronJob) container "find-miners" does not have a read-only root file system (check: no-read-only-root-fs, remediation: Set readOnlyRootFilesystem to true in the container securityContext.)

@MatousJobanek WDYT ^^^ ?